docs/source/cli.rst

```diff
@@ -31,7 +31,7 @@ Read it
     optional arguments:
       -h, --help            show this help message and exit
       --pattern PATH, -p PATH
-                            paths to pattern definition files
+                            paths to pattern definition files or modules
```

Comment:
@ajm188 No, because right now the documentation says you can give it any path, but actually, only modules that are reachable from `sys.path` will work.